src/platform/OpenThread/GenericNetworkCommissioningThreadDriver.h

```diff
@@ -106,6 +106,7 @@ class GenericThreadDriver final : public ThreadDriver
 
     CHIP_ERROR CommitConfiguration() override;
     CHIP_ERROR RevertConfiguration() override;
+    CHIP_ERROR DisconnectFromNetwork() override;
```

Comment:
Nit: putting this after `ConnectNetwork` below would be a more logical grouping.

Reply:
Done.